apps/docs/src/guide/introduction/creating-a-fuel-dapp.md

```diff
@@ -0,0 +1,179 @@
+# Creating a Fuel dApp
+
+`npm create fuels` is a command line tool that helps you scaffold a new fullstack Fuel dApp. In this guide, we will create a new counter dApp using `npm create fuels` and add decrement functionality to it. The final result will look like this:
+
+![End result of this guide](../../public/creating-a-fuel-dapp-create-fuels-end-result.png)
+
+## Initializing the project
+
+The first step is to run the command:
+
+::: code-group
+
+```sh [npm]
+npm create fuels
+```
+
+```sh [pnpm]
+pnpm create fuels
+```
+
+:::
+
+Once you run the command, you will be greeted with a few simple questions. We will answer them as follows:
+
+```md
+◇ What is the name of your project?
+│ my-fuel-project
+│
+◇ Select a package manager:
+│ pnpm
+│
+◆ Which Sway programs do you want? (space to toggle)
+│ ● Contract
+│ ○ Predicate
+│ ○ Script
+└
+```
+
+The tool will scaffold the project and install the necessary dependencies for you. You will then be greeted with this message:
+
+```md
+⚡️ Success! Created a fullstack Fuel dapp at my-fuel-project
```

Comment:
```suggestion
⚡️ Success! Created a fullstack Fuel dApp at my-fuel-project
```

Reply:
I'm afraid I'll have to make this change in the CLI tool's code in a separate PR when I update those docs links because I have just copied over the exact message that the tool logs out when you run it.

Reply:
#2016